x-pack/plugins/reporting/public/share_context_menu/register_pdf_png_reporting.tsx

```diff
@@ -106,7 +106,11 @@ export const reportingScreenshotShareProvider = ({
       return [];
     }
 
-    if (isSupportedType && !capabilityHasVisualizeScreenshotReporting) {
+    if (
+      isSupportedType &&
+      !capabilityHasVisualizeScreenshotReporting &&
+      !capabilityHasDashboardScreenshotReporting
```

Comment:
We don't want to return an empty array for users Dashboard if they have the right permissions defined in isSupportedType